src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -201,7 +196,8 @@ public override IEnumerable<DownloadClientItem> GetItems()
 
             if (ignoredCount > 0)
             {
-                _logger.Warn("{0} torrent(s) were ignored because they did not have a title. Check Deluge and remove any invalid torrents");
+                _logger.Warn("{0} torrent(s) were ignored because they did not have a hash or title. Check Deluge and remove any invalid torrents");
```

Comment:
Sorry, I don't know how to do a fancy "replace this with that" comment, but `ignoredCount` seems to be missing. (Was already missing in the past)

Reply:
@zakkarry based on the warning never showing the count maybe no one ever looked at it 😆

Reply:
lol that tracks

Reply:
> Sorry, I don't know how to do a fancy "replace this with that" comment, but `ignoredCount` seems to be missing. (Was already missing in the past)

You'd use
````
```suggestion
what-to-replace-with
```
````